lib/compiler/doc/src/compile.xml

```diff
@@ -386,6 +386,14 @@ module.beam: module.erl \
             </list>
           </item>
 
+          <tag><c>{include_lib_paths,[{'application', "RootDir"}]}</c></tag>
```

Comment:
Would it make sense to use `"AppDir"` since "root dir" already has implications regarding OTP? See for example:

```
9> code:root_dir().
"/home/ferd/.erln8.d/otps/OTP-21.0/dist/lib/erlang"
```

Reply:
Good point, thanks!